### ISSUE TITLE
Current EnergyUse Sensory reporting incorrectly.

### ISSUE THREAD

**Issue by stumbows** [petretiandrea/home-assistant-tapo-p100#247]
Currently running v1.2.14 HACS integration.

Devices related to this issue are Tapo P110.

Issue occurs when wattage in use crosses 1000W. Current power is shown in kWh even though sensor is reading in Watts. 

What happens is when the usage gets higher than 1,700w it reports as 1.8w instead of 1.8kW. This is giving me really inconsistent readings.

The tapo_sensor_entity.py shows the following for this sensor.

```
class TapoCurrentEnergySensor(TapoSensor):
    def __init__(self, coordiantor: TapoUpdateCoordinator):
        super().__init__(
            coordiantor,
            SensorConfig(
                "current energy",
                DEVICE_CLASS_POWER,
                STATE_CLASS_MEASUREMENT,
                POWER_WATT,
            ),
        )

    @property
    def native_value(self) -> StateType:
        data: TapoDeviceState = self.coordinator.data
        if data.energy_info is not None:
            return data.energy_info.current_power / 1000
        return None
```

I think the divide by 1000 may be causing this but confused why it only happens when the usage gets much higher.

**Comment by novotnej** [comment]
I believe this may have been broken by the latest firmware update - when I reload the device, the currentEnergy updates correctly, but then never changes until I reload the device again. 

I have multiple P110, one that is still on the old firmware and that one is showing up in HA just fine.

**Comment by petretiandrea** [comment]
Closed by #280
